include/rocksdb/rate_limiter.h

```diff
@@ -40,6 +40,14 @@ class RateLimiter {
   // REQUIRED: bytes_per_second > 0
   virtual void SetBytesPerSecond(int64_t bytes_per_second) = 0;
 
+  // This API allows user to dynamically change rate limiter's refill period
+  // (microsecond)
+  //
+  // REQUIRED: refill_period_us > 0
+  virtual Status SetRefillPeriodUs(int64_t /* refill_period_us */) {
```

Comment:
How about SetSingleBurstBytes(int64_t burst_bytes)? Then the `RateLimiter` API won't have a new concept (refill period) as it can be derived from `burst_bytes` when needed.

Reply:
Sure!

Reply:
Fixed